dorpsgek_github/__main__.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import http
+import sys
+
+from aiohttp import web
+
+from dorpsgek_github import github
+from dorpsgek_github.config import GITHUB_APP_PORT
+
+# List of modules that are enabled
+from dorpsgek_github.modules import (  # noqa
+    ping,
+    registration,
+)
+
+
+log = logging.getLogger(__name__)
+
+
+async def github_handler(request):
+    headers = request.headers
+    data = await request.read()
+
+    try:
+        await github.process_request(headers, data)
+        return web.Response(status=http.HTTPStatus.OK)
+    except Exception:
+        log.exception("Failed to handle GitHub event!")
+        return web.Response(status=http.HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+async def github_startup(app):
+    await github.startup()
+
+
+def main(args):
+    app = web.Application()
+    app.router.add_post("/", github_handler)
+    app.on_startup.append(github_startup)
+
+    web.run_app(app, port=GITHUB_APP_PORT)
+
+
+if __name__ == '__main__':
+    main(sys.argv[1:])
```

Comment:
Don't see any indication of what `sys.argv[1:]` is supposed to be? is it optional?

Reply:
(and currently we don't do anything with the command-line; but the entry_point forces us to have it as parameter to `main()`, so we might as well pass it along if you call this module with `python -m dorpsgek_github`.

Reply:
Meh; I remembered the wrong way around, and `entry_point` does NOT add it. So I removed it completely :D